opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -188,6 +188,12 @@ message Metric {
     ExponentialHistogram exponential_histogram = 10;
     Summary summary = 11;
   }
+
+  // Additional attributes that describe the metric. [Optional].
+  // Attributes are non-identifying.
+  // Attribute keys MUST be unique (it is not allowed to have more than one
+  // attribute with the same key).
+  repeated opentelemetry.proto.common.v1.KeyValue attributes = 12;
```

Comment:
What's the reason this field is here and not in the `*DataPoint` messages?

Reply:
`*DataPoint` messages already contain attributes, so a second set of attributes seems a bit odd.  More importantly, we are interested describing things about the metric, or aggregation (e.g. gauge, sum, etc).  My primary use case, representing the "prometheus type", is defined for an entire prometheus `MetricFamily`, which corresponds to the `Metric` in OTel.

Reply:
Make sense.